Seth-Daily/Seth-Daily.md

```diff
@@ -21,14 +21,14 @@ quoteAuthor: "Mark Twain"
 
 Seth is an SSW Marketing Assistant and Account Manager who loves artificial intelligence and applying it for productivity.
 
-⌛ He's always coming up with time-saving tools using ChatGPT and other AI services, and stays up to date on the latest in the AI world.
+⌛ He's always coming up with time-saving tools using ChatGPT and other AI, and follows AI world closely.
 
-📏 He likes improving [SSW Rules](https://www.ssw.com.au/rules/) because best practice standards are the fastest way to the optimization of every process.
+📏 He likes improving [SSW Rules](https://www.ssw.com.au/rules/) because best practice standards optimize every process.
 
-🎊 He contributes to the [SSW Website](https://www.ssw.com.au/), organizes [events](https://www.ssw.com.au/ssw/Events/?tech=all&type=all) and is learning account management skills under the guidance of [Uly](https://www.ssw.com.au/people/uly/).
+🎊 He contributes to the [SSW Website](https://www.ssw.com.au/), organizes [events](https://www.ssw.com.au/ssw/Events/?tech=all&type=all) and is learning account management under the guidance of [Uly](https://www.ssw.com.au/people/uly/).
 
-🌎 Originally from California, he lived for a stint in Switzerland and now has a second home in Australia.
+🌎 Raised in California, he has lived in Switzerland and now his second home is Australia.
```

Comment:
isn't it your 3rd home?

Reply:
No I guess the second place wasn't a home only the third but that's confusing
Done - fixed